spec/controllers/physical_chassis_controller_spec.rb

```diff
@@ -0,0 +1,70 @@
+require './app/controllers/physical_chassis_controller'
```

Comment:
Hm, usually you don't need to require the controller itself, were the tests failing without this line?

Reply:
Hi @skateman ! I am going to update the screenshot. :+1: 

About the *require*, the tests were failing. I couldn't see why the constant *PhysicalChassisController* is not being autoloaded from the UI Controller.

Do you have any previous experience with this? The only way that the controller could be referenced was with the explicit require. :cry:

Reply:
@caiocmpaes sure, let me take a quick look

Reply:
Hm, it seems working for me ... can you try to push without the require? I think travis will pass green.

Reply:
Ok. I'll do that right now. Thank you!